### DISCUSSION TITLE
Changing properties in Azure AD synced environment

### DISCUSSION THREAD

**Discussion by ehlen000** [KelvinTegelaar/CIPP#1397] [Q&A] [answered]
Tried to change a UPN name in a Azure AD synced environent. This functionality is greyed out in the Azure AD portal, but CIPP accepts the change, processes it, and then the Azure AD sync breaks (as expected) and the user cannot login anymore. We repaired this using powershell. 

We want to give CIPP functionality to our first line support, but this is a risk because the power of CIPP can make them very vularable to mistakes. 

Is this intented behaviour? or should these fields be greyed out in CIPP? 

Kind regards,
Andrew

**Comment by KelvinTegelaar** [comment, marked as answer]
Intended behaviour, you can do the same in the M365 portals.
